deploy/helm/sumologic/values.yaml

```diff
@@ -4125,6 +4129,212 @@ metadata:
       ## To use maxUnavailable, set minAvailable to null and uncomment the below:
       # maxUnavailable: 1
 
+## Configure log collection with Otelcol
+## This is currently highly experimental and not recommended unless you really know what you're doing
+## In particular, this configuration currently only covers container logs, and only supports docker-shim
+## This is an alpha feature, and may change in the near future
+logs:
+  enabled: false
```

Comment:
Now that I look at this key it seems to me it will be too misleading. Let's change the whole key from `logs` into `otelcollogs` (for lack of better ideas) for the time being 🙏

Reply:
I'd argue not to to something "just for now".

It will be a pain and unnecessary effort later on to migrate/change this

Reply:
It is an alpha feature as is, so we can change it without breaking changes. it'd be nice to have a clear vision of how we want this to look in 3.0, though.

Reply:
Changed to `otellogs`, it's consistent with `otelagent`, and `otelcollogs` just looks weird to me.

Reply:
Thank you!